outsettingsvalues/tests.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test outsettingsvalues view.
+"""
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.shortcuts import reverse
+from django.test import Client, RequestFactory, TestCase
+
+from outsettingsvalues.views import BaseView
+
+
+class TestBaseView(TestCase):
+    """
+    BaseView test class.
+    """
+
+    def setUp(self):
+        """
+        Create mane instances for all test.
+
+        :return: None
```

Comment:
Do we really need documentation about function result (what it returns) here and below?